src/managed/OpenLiveWriter.HtmlEditor/CodeHighlighter.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+
+namespace OpenLiveWriter.HtmlEditor
+{
+    public class CodeHighlighter
+    {
+        private const int TAB_SIZE = 4;
+
+        private const string PRE_OPEN = "<pre>";
+        private const string PRE_CLOSE = "</pre>";
+        private const string PRE_OPEN_STYLED = "<pre class=\"prettyprint\">";
+        private const string PRETTIFY_SCRIPT_NAME = "run_prettify.js";
+        private const string DEFAULT_SKIN = CodeHighlighterSkins.Sunburst;
+
+        private static readonly string _prettifyScript =
+            $"<script src=\"https://cdn.rawgit.com/google/code-prettify/master/loader/{PRETTIFY_SCRIPT_NAME}?skin={DEFAULT_SKIN}\"></script>";
```

Comment:
https://rawgit.com/ is currently in sunset and will be shut-down by October. Whatever service is used instead, the user should be informed that a third-party script is being inserted into their site, and a third-party service is used to fetch it. Use of [subresource integrity](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) would also be desired.

Reply:
I found this site that produces HTML for pretty printing code, and it has an API you can call:

http://hilite.me/

I've not yet worked out what this PR is actually doing, so I haven't any ideas on how to use it to update this PR, but thought I'd mention it in case some one else can use it to update this PR, or else explain what this code is doing so that I can attempt to update it.